src/classpath-openjdk.cpp

```diff
@@ -669,6 +669,10 @@ class MyClasspath : public Classpath {
 #  ifdef PLATFORM_WINDOWS
     expect(t, loadLibrary(t, libraryPath, "msvcr100", true, true));
 #  endif
+    // necessary for using OpenJDK builds from
+    // https://github.com/ojdkbuild/ojdkbuild:
+    loadLibrary(t, libraryPath, "ojdkbuild_zlib", true, true);
```

Comment:
Serious edge case: Could this cause problems if the library is there, but we're not supposed to link it?  I.e. Perhaps the user has multiple installs of openjdk, one from ojdkbuild and one not.